tests/unit_tests/arelle/test_version.py

```diff
@@ -0,0 +1,92 @@
+import sys
```

Comment:
Arbitrary line for discussion:

> FYI the tests in CI run with tox in a sdist install env, so an arelle._version module exists in the CI env (the tests also shouldn't fail locally if that file exists).

Reply:
I'm not super sure whats up with the tests, I'm considering not patching `import` because it seems to work differently locally for me then in github actions using tox, but I'll have to get back to you on that. FWIW I tested it locally with/without the file existing and it works for me

Reply:
Could you move the import attempt into a function and mock that function in your test?

```python
def getBuildVersion() -> str | None:
    try:
        import arelle._version
        return arelle._version.version
    except ModuleNotFoundError:
        return None
```

Reply:
Yeah thats a clever fix 👍